src/leaflike/dev/sample_data.clj

```diff
@@ -0,0 +1,31 @@
+(ns leaflike.dev.sample-data
+  "Functions to generate and add dummy bookmarks. To be used for testing
+  and demoing."
+  (:require [leaflike.bookmarks.core :as bm-core]
+            [faker.internet :refer [domain-word]]
+            [faker.lorem :refer [sentences]]
+            [clojure.string :as str]))
+
+(defn gen-bookmark
```

Comment:
This is getting used internally in this ns and likely to stay this way. Shouldn't this be marked as private?